calculator.js

```diff
@@ -0,0 +1,15 @@
+const THEanswer = 2;
+console.log(THEanswer + 40);
+
+const couldbe = 2;
+const maybe = 4 * 10;
+console.log(couldbe + maybe);
+
+const dontknow = 7 + 5;
+const notsure = 2 * 3;
+const okayyes = notsure * 5;
+console.log(dontknow + okayyes);
+
+const again = 8 * 9;
+const yup = - 30;
+console.log(again + yup);
```

Comment:
LOL on these variable names. Just make sure to camelCase multiword variables.

couldBe
dontKnow
...etc